test/integrationTests/codeActionRename.integration.test.ts

```diff
@@ -0,0 +1,38 @@
+/*--------------------------------------------------------------------------------------------- 
+ *  Copyright (c) Microsoft Corporation. All rights reserved. 
+ *  Licensed under the MIT License. See License.txt in the project root for license information. 
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fs from 'async-file';
+import * as vscode from 'vscode';
+
+import poll from './poll';
+import { should, expect } from 'chai';
+import testAssetWorkspace from './testAssets/testAssetWorkspace';
+
+const chai = require('chai'); 
+chai.use(require('chai-arrays')); 
+chai.use(require('chai-fs')); 
+ 
+suite(`Code Action Rename ${testAssetWorkspace.description}`, function() {
+    suiteSetup(async function() { 
+        should();
+
+        let csharpExtension = vscode.extensions.getExtension("ms-vscode.csharp"); 
+        if (!csharpExtension.isActive) { 
+            await csharpExtension.activate(); 
+        }
+
+        await csharpExtension.exports.initializationFinished;
+
+    });
+    
+    test("Code actions can remame and open files", async () => {                
+        let fileUri = await testAssetWorkspace.projects[0].addFileWithContents("test.cs", "class C {}");
+        await vscode.commands.executeCommand("vscode.open", fileUri);
+        let c = await vscode.commands.executeCommand("vscode.executeCodeActionProvider", fileUri, new vscode.Range(0, 7, 0, 7)) as {command: string, arguments: string[]}[];
+        expect(c.length).to.equal(3);
```

Comment:
3 seems like a magic number. Can we provide some text [`expect` should take a second parameter that is used to create the failure output string] that tells us why 3 is the right answer?